### DISCUSSION TITLE
Do V have a vector library

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#20999] [Questions and Answers]
What is the standard vector library for V like? I didn't find this, can anyone tell me if they know?

**Comment by Jengro777** [comment]
https://vlang.github.io/vtl/index.html

Is this？

**Comment by JalonSolov** [comment]
It depends on what you mean by "vector library".

As it says on the link you pasted, that is the V `tensor` library.

**Comment by larpon** [comment]
There's also https://modules.vlang.io/math.vec.html
